src/file_request_handler.cc

```diff
@@ -73,7 +75,7 @@ static bool checkFileAndSubtitle(fs::path& path, const std::shared_ptr<CdsObject
         }
         throw_std_runtime_error("Failed to open {}: {}", path.c_str(), std::strerror(errno));
     }
-    return is_srt;
+    return { is_srt, mimeType, statbuf };
```

Comment:
That copies the whole struct which results in a similar issue as https://github.com/gerbera/gerbera/security/code-scanning/398?query=ref%3Arefs%2Fheads%2Fmaster

Reply:
dead link for me.

Reply:
![grafik](https://user-images.githubusercontent.com/23654788/130430923-67566f0a-eb13-418b-8973-b2ccfdd2ea90.png)

Reply:
Out of curiosity, does that check trigger elsewhere?

Reply:
> Out of curiosity, does that check trigger elsewhere?

CodeQL triggered at the new struct which is in the screenshot. Nowhere else, right now.

Reply:
Strange that this PR didn't touch that code.